### PR TITLE
feat(edgeless): note block supports auto complete panel

### PR DESCRIPTION
**Pull request by fundon**
Closes: BS-97, BS-96, BS-89

https://github.com/toeverything/blocksuite/assets/27926/f75b0222-9962-4107-9c46-6668a9dcbc81

https://github.com/toeverything/blocksuite/assets/27926/27b0f2ad-b86e-4eb2-b4b2-7f51d365142f

https://github.com/toeverything/blocksuite/assets/27926/456387bb-f24a-4ecb-a4eb-58728cbfb84e

### CODE REVIEW

**Comment by fundon** [comment]
* **#6591** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/6591?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/root-block/edgeless/components/auto-complete/edgeless-auto-complete.ts [hunk 1]
```diff
@@ -258,10 +264,23 @@ export class EdgelessAutoComplete extends WithDisposable(LitElement) {
 
   private _addConnector(source: Connection, target: Connection) {
     const { service } = this.edgeless;
+
+    let strokeColor = '--affine-palette-line-grey';
```
Comment (fundon) [inline packages/blocks/src/root-block/edgeless/components/auto-complete/edgeless-auto-complete.ts:268]: `--affine-palette-line-grey`  `--affine-tag-gray`

Can we standardize the naming of these?

**Comment by hwangdev97** [comment]
if not matched, i think we need use grey/gray color, not black 🤔

**Comment by donteatfriedrice** [comment]
> if not matched, i think we need use grey/gray color, not black 🤔

Maybe we not inherit the width and height ratio of note? The shape looks a bit weird 🤔.

**Comment by fundon** [comment]
> Maybe we not inherit the width and height ratio of note? The shape looks a bit weird 🤔.

Yes, the width and height need some adjustment.

**Review by Saul-Mirone — CHANGES_REQUESTED** [review]
Please add tests.

**Review by Saul-Mirone — APPROVED** [review]

**Pull request merged**